src/main/index.ts

```diff
@@ -70,7 +70,8 @@ app.on('activate', () => {
   }
 })
 
-const CONFIG_DIR = isDevelopment ? path.join(__dirname, '../../config') : path.join(__dirname, '../../../config')
+// Absolute path for configs -> /AppData/Local/omniconfig
+const CONFIG_DIR = isDevelopment ? path.join(__dirname, '../../config') : path.join(app.getPath('home'), 'AppData/Local/omniconfig')
```

Comment:
https://www.electronjs.org/docs/api/app#appgetpathname

Consider using `app.getPath('appData')` instead of prepending home to stuff